src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -297,11 +297,11 @@ private void readAndUpdateMutesRcas() {
 
         LOG.info("Updating the muted RCA Graph to : {}", rcasForMute);
         Stats.getInstance().updateMutedGraphNodes(rcasForMute);
-      } catch (Exception e) {
-        LOG.error("Couldn't read/update the muted RCAs.", e);
       }
+      lastModifiedTimeInMillisInMemory = lastModifiedTimeInMillisOnDisk;
+    } catch (Exception e) {
+        LOG.error("Couldn't read/update the muted RCAs.", e);
```

Comment:
You might want to emit a metric here. If we really expect never for this to happen, then we should emit a domain level metric and add it to the dashboard.